common/djangoapps/student/forms.py

```diff
@@ -203,15 +206,9 @@ class AccountCreationForm(forms.Form):
             "max_length": _("Email cannot be more than %(limit_value)s characters long"),
         }
     )
-    password = forms.CharField(
-        min_length=password_min_length(),
-        max_length=password_max_length(),
-        error_messages={
-            "required": _PASSWORD_INVALID_MSG,
-            "min_length": _PASSWORD_INVALID_MSG,
-            "max_length": _PASSWORD_INVALID_MSG,
-        }
-    )
+
+    password = forms.CharField()
```

Comment:
This is the old non-logistration view, right? (The registration view that comes default with OpenEdx - whereas edx.org uses the new-style "logistration" combined view.)

In that case, the client javascript code needs these settings to be there to do any validation at all. That javascript code currently does not call our python endpoint to do the validation for it.

It's fine to remove these settings on the CharField, if you also fix the frontend code to call our validation endpoint (which would be a great fix, honestly - Cali was running up against that oddity in her unicode normalization work).